controller/src/beerocks/bml/bml.h

```diff
@@ -648,6 +648,51 @@ int bml_get_dcs_scan_results(BML_CTX ctx, const char *radio_mac,
 int bml_start_dcs_single_scan(BML_CTX ctx, const char *radio_mac, int dwell_time_ms,
                               int channel_pool_size, unsigned int *channel_pool);
 
+/**
+ * Get client list.
+ *
+ * @param [in] ctx                  BML Context.
+ * @param [out] client_list         Client list.
```

Comment:
this is actually an `[in/out]` parameter (the calling user need to provide an already allocated array with the max size provided by the `client_list_size` which is also `[in/out]`
(see bml_get_vap_list_credentials as an example)